desc/compute/_curve.py

```diff
@@ -830,20 +932,22 @@ def _frenet_binormal(params, transforms, profiles, data, **kwargs):
     label="\\kappa",
     units="m^{-1}",
     units_long="Inverse meters",
-    description="Scalar curvature of the curve",
+    description="Scalar curvature of the curve, positive/negative = concave/convex",
```

Comment:
what is concave/convex in relation to here? Might be clearer to say whether the curve is curving towards the center or away?

Reply:
or maybe pictorially:
`*center   )+curvature    (-curvature`